cli/src/maintenance.rs

```diff
@@ -793,6 +870,7 @@ pub fn try_perform_maintenance(
         // ensure that the balance update needs to reference as few accounts
         // as possible.
         .or_else(|| state.try_merge_on_all_stakes())
+        .or_else(|| state.try_unstake_from_inactive())
```

Comment:
I would put this after updating the exchange rate; we should update the exchange rate as soon as we can, but removing the validator is not high-prio; it can’t receive new stake anyway.